src/Core/GL/GLRenderable.hpp

```diff
@@ -36,15 +40,19 @@ class GLRenderable
 	// TODO: This should be all part of the RenderModel's Primitive!
 	// A mesh consists basically just out of a contiguous range of triangles
 
-	// Vertices stored in a linear vector, every 3 floats -> one vertex, easy geometry submission.
+	/**
+	 * Vertices stored in a linear vector, every 3 floats -> one vertex, easy geometry submission..
+	 */
 	std::vector<float> vertices;
 
-	// A vertex has a stride of 3 floats (x,y,z).
+	//A vertex has a stride of 3 floats (x,y,z).
```

Comment:
Try to avoid unnecessary changes. Turn into a short documentation comment?

Reply:
This was added so doxygen would show the docstring on the website.